### ISSUE TITLE
Rubocop error on safe navigation operator

### ISSUE THREAD

**Issue by cmitz** [sds/slim-lint#71]
In my template, I want to use the relatively new safe navigation operator like this:

```
= @some_object&.name
```

Since the object might be nil, so I use the [safe navigation operator](http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/) to prevent errors. However, rubocop errors on that line with `Rubocop: Lint/Syntax: unexpected token error`, making Atom warn me on normally valid syntax. 

I'm not sure if it is because of Rubocop or slim_lint, but normally Rubocop doesn't crash on the `&`, so that's why I'm listing it here

**Comment by sds** [comment]
Hey @cmitz, which Ruby version is your RuboCop targeting? Make sure it's 2.3 or later.

```yaml
# rubocop.yml
AllCops:
  TargetRubyVersion: 2.3
```

If you haven't specified the target Ruby version, RuboCop will by default use the `.ruby-version` file in your repo to determine which Ruby version to syntax check with.

If you're still having issues, would appreciate a reproducible example!

**Comment by cmitz** [comment]
We're using Ruby 2.4.2, so that shouldn't be an issue. I will make a demo project available for you to see what happens. It might take a day or two for me to find a moment where I can do that.

**Comment by cmitz** [comment]
Ok, that took a lot longer than expected, but the issue is demonstrated here:

https://github.com/cmitz/slim_safe_operator

**Comment by sds** [comment]
Hey @cmitz, thanks for taking the time to put together reproducible test case.

As I [mentioned before](https://github.com/sds/slim-lint/issues/71#issuecomment-338834501), you need to explicitly specify the version of Ruby you are targeting in your RuboCop configuration. When I run `slim-lint` (0.15.1) on Ruby 2.4.2 against your repository:

```
$ git clone git@github.com:cmitz/slim_safe_operator.git
...
$ cd slim_safe_operator
$ ruby -v
ruby 2.4.2p198
$ slim-lint .
~/slim_safe_operator/app/views/index/index.html.slim:13 [W] RuboCop: unexpected token error
(Using Ruby 2.1 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)
```

This is fixed by either:

* Adding a `.ruby-version` file containing `2.4.2` (the version of Ruby you are running) to the repository so that RuboCop automatically targets that Ruby version when parsing
* Adding a `.rubocop.yml` file that explicitly targets Ruby 2.3 or newer:
  ```yaml
  AllCops:
    TargetRubyVersion: 2.3
  ```

When I do either of the above, I no longer get errors. Hope this helps!

**Comment by cmitz** [comment]
Oh, sorry for wasting your time then :)

I read `TargetRubyVersion` as `TargetRailsVersion` (what I had in my real project). That's probably because I was staring at a screen for too long.
